src/rbd.cc

```diff
@@ -505,7 +509,7 @@ static int do_list(librbd::RBD &rbd, librados::IoCtx& io_ctx, bool lflag,
     cout << tbl;
   }
 
-  return 0;
+  return r;
```

Comment:
NIT: I'd feel better if this said return r < 0 ? r : 0;

In the current code it doesn't make any difference, but I'd like to be guaranteed that this returns 0 on success as before.